hlxsites/merative2
# fix(leadspace-document): increase padding bottom to account for the overlap

## Issue

Fixes - N/A

## Description

**Changed**

- This PR adds a fix to a regression styling issue that might have gotten introduced by this PR (https://github.com/hlxsites/merative2/pull/309).
- The PDF viewer page has a slight overlap treatment, which requires a different padding-bottom value that's not the same as the top.

## Design Specs

- Figma Link - N/A

## Test URLs

![Screenshot 2023-08-11 at 9 58 48 AM](https://github.com/hlxsites/merative2/assets/1815714/a826f16d-1863-4474-8ac0-f4ffe1041290)

- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/documents/brief/ecoa-epro-data-sheet

![Screenshot 2023-08-11 at 10 00 41 AM](https://github.com/hlxsites/merative2/assets/1815714/466c9569-3503-46e7-bdbb-4007e977f305)

- After (Changes from this PR): https://pdf-viewer--merative2--hlxsites.hlx.page/documents/brief/ecoa-epro-data-sheet

## Testing Instruction

- Page without the PDF download button: Defaults to INLINE Mode and the PDF viewer frame is visible on mobile
  - Visit this draft page (https://pdf-viewer--merative2--hlxsites.hlx.page/drafts/putra/payers-weight-loss-drugs)
  - Ensure that the inline embed mode is being displayed on mobile
  - Ensure that there's enough padding-bottom spacing within the leadspace on all breakpoints.


- Page with PDF down button: Defaults to the frame mode on desktop, but hidden on mobile
  - Visit this page (https://pdf-viewer--merative2--hlxsites.hlx.page/documents/brief/ecoa-epro-data-sheet)
  - Ensure that the PDF viewer is not visible on Mobile and Tablet, just like this [inspiration site ](https://www.purestorage.com/docs.html?item=/type/pdf/subtype/doc/path/content/dam/pdf/en/solution-briefs/sb-pure-technical-support.pdf)
  - Ensure that there's enough padding-bottom spacing within the leadspace on all breakpoints.
